src/basic/strxcpyx.c

```diff
@@ -19,8 +19,8 @@
 
 /*
  * Concatenates/copies strings. In any case, terminates in all cases
- * with '\0' * and moves the @dest pointer forward to the added '\0'.
- * Returns the * remaining size, and 0 if the string was truncated.
+ * with '\0' and moves the @dest pointer forward to the added '\0'.
+ * Returns the remaining size, and 0 if the string was truncated.
```

Comment:
The comment seems to be a bit out of date, so it is probably should be updated or removed.